client-app/shared/cart/components/cart-line-items.vue

```diff
@@ -35,6 +35,7 @@
 
         <VcInStock
           :quantity="item.inStockQuantity"
+          :is-track-inventory="item.availabilityData?.isTrackInventory"
```

Comment:
Where is no such prop

Reply:
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/6369252/591d8322-deef-4bc9-8c67-71d0f27e6d7a)